src/scripting/custom_particles.cpp

```diff
@@ -376,11 +359,10 @@ void CustomParticles::ease_lifetime(float lifetime, float time, std::string easi
   object.ease_value(&object.m_particle_lifetime, lifetime, time, getEasingByName(EasingMode_from_string(easing)));
 }
 
-
-
 // =============================================================================
-//   lifetime_variation
+//   Lifetime_variation
```

Comment:
All such comments in this file should avoid the underscore, since it doesn't really represent any variable.

For example, this should be "Lifetime Variation".